File: sdks/thunkmetrc-go/wrapper/models/facility.go
package models

type Facility struct {
    Alias *string `json:"Alias,omitempty"`
    AllowTransferOfOnHoldPackages bool `json:"AllowTransferOfOnHoldPackages,omitempty"`
    AllowTransferOfOnRecallPackages bool `json:"AllowTransferOfOnRecallPackages,omitempty"`
    CredentialedDate string `json:"CredentialedDate,omitempty"`
    DisplayName string `json:"DisplayName,omitempty"`
    Email *string `json:"Email,omitempty"`
    FacilityId int64 `json:"FacilityId,omitempty"`
    FacilityType FacilityFacilityType `json:"FacilityType,omitempty"`
    HireDate string `json:"HireDate,omitempty"`
    IsFinancialContact bool `json:"IsFinancialContact,omitempty"`
    IsManager bool `json:"IsManager,omitempty"`
    IsOwner bool `json:"IsOwner,omitempty"`
    License FacilityLicense `json:"License,omitempty"`
    Name string `json:"Name,omitempty"`
    Occupations []interface{} `json:"Occupations,omitempty"`
    SupportActivationDate *string `json:"SupportActivationDate,omitempty"`
    SupportExpirationDate *string `json:"SupportExpirationDate,omitempty"`
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/Facility.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class Facility(
    @JsonProperty("Alias")
    val alias: String? = null,
    @JsonProperty("AllowTransferOfOnHoldPackages")
    val allowTransferOfOnHoldPackages: Boolean? = null,
    @JsonProperty("AllowTransferOfOnRecallPackages")
    val allowTransferOfOnRecallPackages: Boolean? = null,
    @JsonProperty("CredentialedDate")
    val credentialedDate: String? = null,
    @JsonProperty("DisplayName")
    val displayName: String? = null,
    @JsonProperty("Email")
    val email: String? = null,
    @JsonProperty("FacilityId")
    val facilityId: Int? = null,
    @JsonProperty("FacilityType")
    val facilityType: FacilityFacilityType? = null,
    @JsonProperty("HireDate")
    val hireDate: String? = null,
    @JsonProperty("IsFinancialContact")
    val isFinancialContact: Boolean? = null,
    @JsonProperty("IsManager")
    val isManager: Boolean? = null,
    @JsonProperty("IsOwner")
    val isOwner: Boolean? = null,
    @JsonProperty("License")
    val license: FacilityLicense? = null,
    @JsonProperty("Name")
    val name: String? = null,
    @JsonProperty("Occupations")
    val occupations: List<Any>? = null,
    @JsonProperty("SupportActivationDate")
    val supportActivationDate: String? = null,
    @JsonProperty("SupportExpirationDate")
    val supportExpirationDate: String? = null
)
